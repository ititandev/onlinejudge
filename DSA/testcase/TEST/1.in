10
-50.08 -79.71 31.37 46.21 80.09 4.37 -49.18 -42.57 -30.18 51.74
