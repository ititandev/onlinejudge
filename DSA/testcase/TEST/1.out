  -79.71  -50.08  -49.18  -42.57  -30.18
    4.37   31.37   46.21   51.74   80.09
